### DISCUSSION TITLE
What is the future of Quasar?

### DISCUSSION THREAD

**Discussion by ivanjaros** [quasarframework/quasar#17726] [General - Components / Directives / etc] [answered]
I have observed incredible decrease of activity in Discord in the past years. Github issues seem to be inactive. Discussions here as well. It just looks to me like this project is slowly dying. Not sure if it is lack of core devs or users themselves. It's just strange, because I very well remember how active the Discord used to be, for example. Nowadays its like a graveyard. This concerns me also from business POV since I was looking forward to use Quasar in production(finally) and not just for personal things. But now, I am not sure if there is future here.

**Comment by rstoenescu** [comment, marked as answer]
Every year or so, there’s a message like this for every framework. For us, it always appears after a major release (in this case two! major CLI releases), after a major effort on a new feature, after a docs improvement/rewrite etc etc. So this message is like clockwork. Was wondering when it is going to show up :) The more we work on the framework, the more frequent this kind of messages pop up. Open source is fun!

TLDR: We’re here to stay. We’re not going anywhere.

**Reply by ivanjaros** [reply]
I'm glad to hear it, from commercial point of view. But the massive outflow of users cannot be overlooked. Not saying it's just quasar, I kinda observed weird behavior since 2020 with massive outflow of users in online communities overall.

**Reply by zanzara** [reply]
In my opinion the decrease of activity on discord and on github-discussion/issues
began with chatGPT and successors as copilot and other AI.
Why wait for an answer/reply minutes/hours/days or even weeks, if one can get it in a wink?!

But the thing with Quasar is (aswell only my opinion),  the missing outlook in the roadmap.
That's a very important indicator in the business area, as I noticed a few times.

**Reply by alexey-vince** [reply]
I agree with @zanzara here - roadmap was not updated for a while (Q3 2024), communication with community with regards to what is next is also missing. So for any new developer considering Quasar the project looks like on the way out.

**Reply by Creatozz** [reply]
Lower activity die to AI help makes sense and also points to the importance of having AI tools easily access Quasar docs. I see there is a GPSs focussed on Quasar which claims to have access to latest docs. How can Quasar support AI to access knowledge so than devs can be sure that new AI tools know very well how to help coding in Quasar?
